releasenotes/notes/new-openSUSE-image-for-SHA256-signing-6e379784502c14bb.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    RPMs on openSUSE are now built with RPM 4.14.3 and have SHA256 digest headers.
```

```suggestion
    SUSE RPMs are now built with RPM 4.14.3 and have SHA256 digest headers.
```

The RPMs are not just for openSUSE, but also for SLES.